pair-programming/week-7/react-forms/App.js

```diff
@@ -1,3 +1,4 @@
+// filepath: week-7/react-forms/App.js
 import { useState } from "react";
```

Comment:
Task 2 looks like its already been done in these files: 
- the `import { useState } from "react";` is already present at the top of the file, replace with instructions
```suggestion
// Import state from React to control the form component
```